pages.it/common/git-reset.md

```diff
@@ -0,0 +1,33 @@
+# git reset
+
+> Annulla commit o rimuovi modifiche dall'area di stage, reimpostando l'HEAD corrente su uno specifico stato.
+> Se viene fornito un percorso, il comando reset si interpreta come "rimuovi dall'area di stage"; se viene fornito l'hash di un commit o un ramo, si interpreta come "annulla commit".
+> Maggiori informazioni: <https://git-scm.com/docs/git-reset>.
+
+- Rimuovi tutto dall'area di stage:
+
+`git reset`
+
+- Rimuovi dall'area di stage uno o più file:
+
+`git reset {{percorso/al/file_1}} {{percorso/al/file_2}}`
```

Comment:
We use this format:
```suggestion
`git reset {{percorso/al/file1}} {{percorso/al/file2}}`
```

Reply:
Thanks! We might then consider updating the few commands that used my same wrong format (bedtools, gixy, git-diff, pass, ...)